recipe/meta.yaml

```diff
@@ -1,47 +1,44 @@
 {% set name = "spyder-kernels" %}
 {% set version = "2.4.0" %}
-{% set hash = "3bd32fbe30a150ab9ccedbd5beb6ddd38bde1d9a8778da52f7edf420bbfb8e51" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spyder-kernels-{{ version }}.tar.gz
+  sha256: 3bd32fbe30a150ab9ccedbd5beb6ddd38bde1d9a8778da52f7edf420bbfb8e51
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 1
+  noarch: python
+  string: "unix_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"  # [unix]
+  string: "win_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"   # [win]
 
 requirements:
-  build:
-    - python                                # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-
   host:
+    - python >=3.0
```

Comment:
Shouldn't this be `3.8` now that Conda-forge doesn't support older versions?

Reply:
That is only for compiled packages. Noarch packages can support anything where conda is available.

Reply:
These packages will only be available via the conda-forge channel for python3.8+, so it is redundant.

Reply:
I think it's still needed, only when https://github.com/conda/conda-build/pull/4606 is released it should be fine.

Reply:
Ok, but this package doesn't work for older Python versions. So could you change this line to `python >=3.7`, and also the one in the run section? After that, this PR would be ready for me.

Reply:
Ok done